src/sentry/conf/server.py

```diff
@@ -329,6 +329,7 @@
     'social_auth.backends.pipeline.user.update_user_details',
     'social_auth.backends.pipeline.misc.save_status_to_session',
 )
+SOCIAL_AUTH_REVOKE_TOKENS_ON_DISCONNECT = True
```

Comment:
i couldnt figure out what the deafult was, but we def should revoke on disconnect